app/src/org/commcare/android/view/EntityView.java

```diff
@@ -442,7 +442,7 @@ private static void removeSpans(Spannable raw) {
     }
 
     private int[] calculateDetailWidths(int fullSize) {

```

Comment:
Even though this is private it would be nice to have method documentation.

After reading this method's code I'm still curious about: 
 - what the entries in the returned array represent?
 - what indeterminate columns are?
 - where hints come from and what type of hint parsing is going on to calculate measurements?